v_0_1_0/master_template.go

```diff
@@ -1723,6 +1723,7 @@ coreos:
       --insecure_bind_address=0.0.0.0 \
       --insecure_port={{.Cluster.Kubernetes.API.InsecurePort}} \
       --kubelet_https=true \
+      --kubelet-preferred-address-types=InternalIP \
```

Comment:
How does the API server know about the internal IP?

Reply:
kubelet register itself with that info